# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mplayer/hotfix.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mplayer/libmpdemux/demux_lavf.c.vanilla	2007-04-18 14:11:07.598629118 +0200
+++ mplayer/libmpdemux/demux_lavf.c	2007-04-18 14:11:36.740289807 +0200
@@ -40,7 +40,7 @@
 #include "avi.h"
 #include "opt.h"
 #endif
-#include "riff.h"
+#include "libavformat/riff.h"
 
 #define PROBE_BUF_SIZE 2048
 
